schemachange/sc_chain.c

```diff
@@ -0,0 +1,20 @@
+#include "sc_chain.h"
+#include "schemachange.h"
+#include "logmsg.h"
+#include <math.h>
+
+void append_to_chain(struct schema_change_type *sc, struct schema_change_type *sc_chain_next){
```

Comment:
Should we reuse bbinc/list.h instead?

Reply:
If that's a list struct, I think no? Because I want each schema change to be able to expand by itself this kind of linked list makes the most sense to me.

Like let's say someone wanted to write a single schema change that first creates a table and then creates an audit trigger on that table. Then it should be able to simply put the table create and audit trigger creates in its sc_chain_next list, and then when it gets to the audit trigger, it will automatically expand it. To me, that makes the most sense with a linked list.

Or maybe I misunderstood the question lol. If so, where can I find bbinc/list.h?

Reply:
But I don't actually end up using a linked list, assides from the simplicity one described by the sc_chain_next field (I had one in a weird spot but didn't end up using it and it is now deleted)